armada_command/command_restart.py

```diff
@@ -44,7 +44,9 @@ def command_restart(args):
                 'Provide more specific container_id or microservice name or use -a/--all flag.'.format(**locals()))
         print('Restarting {instances_count} services {microservice_name}...'.format(**locals()))
     else:
-        print('Restarting service {microservice_name}...'.format(**locals()))
+        microservice_name = instances[0]['ServiceName']
```

Comment:
It shadows microservice_name in line 32 (https://github.com/armadaplatform/armada/pull/3/files#diff-3bd5b70a7be7410cb389c8d49a0890d4L32). I would suggest changing that one to something like microservice_name_or_id.